### PR TITLE
Added more explanation to NFC section. removed outdated browser specific stuff

### PR DESCRIPTION
**Pull request by nitrosimon**

### CODE REVIEW

**Comment by jans23** [comment]
Use English capitalization instead of German capitalization.

File: nitrokey3/troubleshooting.rst [hunk 1]
```diff
@@ -17,50 +17,29 @@ Google and Microsoft Services
...
+
+The backside of the Nitrokey has to be held against the smartphone. For the USB-A Version it might be helpful to lift the side with the USB-Connector slightly to reduce the distance of the end part of the stick. The search of the optimal spot sometimes took up to 20-30 sec.
+
+If you still encounter issues please write a mail at our support. For optimal help please include the shipping/order number and your smartphone model. 
+
 
 
 
 
 update.nitrokey.com does not work:
```
Comment (daringer) [inline nitrokey3/troubleshooting.rst:40]: replace with "The Website update.nitrokey.com Does Not Work:"

english caps

File: nitrokey3/troubleshooting.rst [hunk 2]
```diff
@@ -17,50 +17,29 @@ Google and Microsoft Services
...
-+-------------------+-----------------+-----------------+--------------------------+
-|                   | Chromium 90+    | Arch Packages   |                          |   
-+-------------------+-----------------+-----------------+--------------------------+
-| Mac Os 12         | Firefox 94+     | Firefox.com     |                          |   
-+-------------------+-----------------+-----------------+--------------------------+
-
-We are currently testing more browsers to complete the table.
+There is a bug  with the Key being recognized by the System but a Timeout occurs before the connection succeed. 
 
 NFC is not working
```
Comment (daringer) [inline nitrokey3/troubleshooting.rst:25]: english caps

"NFC is Not Working"

(i know not part of this coomit ;) )

File: nitrokey3/troubleshooting.rst [hunk 3]
```diff
@@ -17,50 +17,29 @@ Google and Microsoft Services
...
 Please make sure FIDO2 is working correctly. Otherwise NFC won't work either.
 
 Also check if you are using the right spot on your smartphone. 
-Find the right spot using: `nfc.fail`_
+Find the right spot using: `nfc.fail`_ 
+
+The backside of the Nitrokey has to be held against the smartphone. For the USB-A Version it might be helpful to lift the side with the USB-Connector slightly to reduce the distance of the end part of the stick. The search of the optimal spot sometimes took up to 20-30 sec.
+
+If you still encounter issues please write a mail at our support. For optimal help please include the shipping/order number and your smartphone model. 
+
```
Comment (daringer) [inline nitrokey3/troubleshooting.rst:37]: Please add the following hints:
* in our experience holding the Nitrokey in a horizontal orientation is better sometimes
* try it without a cover first, then once the right spot is found, retry with a cover

**Review by daringer — CHANGES_REQUESTED** [review]

**Comment by daringer** [comment]
headlines to be reviewed, overall content good, merging

**Pull request merged**